### PR TITLE
Initialise sequencer metrics in a separate function.

### PR DESCRIPTION
**Pull request by pphaneuf**
### Checklist

- [ ] I have updated the [CHANGELOG](CHANGELOG.md).
- [ ] I have updated [documentation](docs/) accordingly (including the [feature implementation matrix](docs/Feature_Implementation_Matrix.md)).

### CODE REVIEW

File: log/sequencer.go [hunk 1]
```diff
@@ -72,25 +72,29 @@ func quotaIncreaseFactor() float64 {
 	return QuotaIncreaseFactor
 }
 
+// InitMetrics sets up the metrics for this package. Must be called before calling IntegrateBatch.
+// Can be called more than once, but only the first call has any effect.
```
Comment (NatalieDoduc) [inline log/sequencer.go:76]: does something still need to be done for the TODO()? 

You're calling your function InitMetrics ), just wanted to check it's not InitJustSomeMetricsButNotAll?

**Review by NatalieDoduc — APPROVED** [review]

File: log/sequencer.go (hunk 1 above)
Comment (pphaneuf) [reply log/sequencer.go:76]: I'm not sure if it is, to be honest? I've done nothing to fix that TODO, and it seems like a reasonable TODO to me.

This is likely to require more PRs to fully address...

Comment (NatalieDoduc) [reply log/sequencer.go:76]: maybe s/the metrics/some metrics/ to avoid confusion.

Comment (pphaneuf) [reply log/sequencer.go:76]: Done

**Pull request merged**